infrastructure/setup_photos_bucket.sh

```diff
@@ -0,0 +1,8 @@
+S3_BUCKET_NAME=nfish-des-kutter-photos
```

Comment:
You could also use input parameters for a script. 

Than you can use the bucket name as input parameter and do not have this duplicated code

Reply:
How do you mean it, to ask the user for it whom executes the script? I don't think that is a good Idea, it should always be the same name for the bucket.

Reply:
or could I use an environment variable in github?